pamelalozano/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,42 +14,62 @@
 
 package com.google.sps.servlets;
 
-import java.util.ArrayList;  
-import java.util.List;  
-import java.util.Date;  
-import com.google.gson.Gson;
+import java.util.ArrayList; 
+import java.util.Date; 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    
+    ArrayList<String> comments = new ArrayList<String>();  
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    
-    Comment prueba = new Comment("Title", "Msg", new Date(System.currentTimeMillis()));
+    response.setContentType("application/json;");
+    response.getWriter().println(comments);
+  }
 
-    /*
-      This array will just be needed once to create the comments, and then it 
-      will not be needed since I'll obtain it from the get request of the comments
-    */
-    ArrayList<Comment> comments = new ArrayList<Comment>();  
-        
-    comments.add(prueba);  
+    @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-    String commentsJson = convertToJsonUsingGson(comments);  
+    // Get the input from the form.
+    Comment newComment = getComment(request);
 
-    response.setContentType("application/json;");
-    response.getWriter().println(commentsJson);
+    //If there's an empty input the subject will be "error"
+    if (newComment.getSubject()=="error") {
```

Comment:
What if someone comments "error"?

Might be better to have a separate method like `validateComment(Comment comment)` that returns a boolean.

Reply:
Oh you are right thank you! I fixed it.